Cargo.toml

```diff
@@ -1,59 +1,60 @@
-iroha_config = { workspace = true }
-futures = "0.3.21"
-http = "0.2.6"
-chrono = { version = "0.4.19", features = ["serde"] }
-hex = "0.4.3"
-parity-scale-codec = "3.2.1"
-thiserror = "1.0"
-rand = { workspace = true, optional = true }
-faker_rand = { workspace = true, optional = true }
-clap = { workspace = true, features = ["derive", "env"] }
-serde = { workspace = true }
-color-eyre = { workspace = true }
-serde_json = { workspace = true }
-
-[features]
-default = ["dev_actor"]
-# Enables an actor that fills Iroha with different data. For dev purposes
-dev_actor = ["dep:rand", "dep:faker_rand"]
-
-[profile.dev.package.backtrace]
-# to optimize color-eyre
-opt-level = 3
+[package]
+name = "iroha2_explorer_web"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+members = [".", "tools/genesis-gen"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[workspace.dependencies]
+iroha_client = { git = "https://github.com/hyperledger/iroha.git", branch='iroha2-dev' }
+iroha_data_model = { git = "https://github.com/hyperledger/iroha.git",branch='iroha2-dev' }
+iroha_crypto = { git = "https://github.com/hyperledger/iroha.git", branch='iroha2-dev'}
+iroha_telemetry = { git = "https://github.com/hyperledger/iroha.git",branch='iroha2-dev'}
+iroha_core = { git = "https://github.com/hyperledger/iroha.git",branch='iroha2-dev'}
+iroha_config = { git = "https://github.com/hyperledger/iroha.git",branch='iroha2-dev' }
+color-eyre = "0.5"
+serde = "1.0"
+serde_json = "1.0"
+clap = { version = "3.1.12", features = ["derive", "env"] }
+rand = { version = "0.8.5" }
+faker_rand = { version = "0.1" }
+
+[dependencies]
+actix = "0.13.0"
+actix-web = { version = "4.0.0", default-features = false, features = ["macros", "compress-brotli", "compress-gzip", "cookies"] }
+awc = { version = "=3.0.0", default-features = false, features = ["compress-brotli", "compress-gzip", "cookies"] }
+actix-server = "2.1.1"
+tracing = "0.1.32"
+tracing-subscriber = { version = "0.3.10", features = ["env-filter"] }
+tracing-log = "0.1.2"
+tracing-actix-web = "0.5.1"
+iroha_client = { workspace = true }
+iroha_data_model = { workspace = true }
+iroha_crypto = { workspace = true }
+iroha_telemetry = { workspace = true }
+iroha_core = { workspace = true }
+iroha_config = { workspace = true }
+futures = "0.3.21"
+http = "0.2.6"
+chrono = { version = "0.4.19", features = ["serde"] }
+hex = "0.4.3"
+parity-scale-codec = "3.2.1"
+thiserror = "1.0"
+rand = { workspace = true, optional = true }
+faker_rand = { workspace = true, optional = true }
+clap = { workspace = true, features = ["derive", "env"] }
+serde = { workspace = true }
+color-eyre = { workspace = true }
+serde_json = { workspace = true }
+url = "2.4.0"
+
+[features]
+default = ["dev_actor"]
+# Enables an actor that fills Iroha with different data. For dev purposes
+dev_actor = ["dep:rand", "dep:faker_rand"]
+
+[profile.dev.package.backtrace]
+# to optimize color-eyre
+opt-level = 3
```

Comment:
```suggestion
opt-level = 3

```

Reply:
Can you please tell me why these commits are required?

Reply:
Umm, not really required, but _usually_ it is _good_ to have a trailing empty newline. I can't say why, but _usually_ code formatters do it.